inc/ThirdParty/Themes/Divi.php

```diff
@@ -172,4 +174,27 @@ public function disable_divi_jquery_body() {
 		}
 
 	}
+
+	/**
+	 * Excludes Divi's CSS files from CSS combination
+	 *
+	 * @since 3.10
+	 *
+	 * @param array $exclude_css An array of CSS  to be excluded.
+	 *
+	 * @return array the updated array of paths
+	 */
+	public function exclude_divi_css_from_combine( $exclude_css ) {
+
+		if ( (bool) $this->options->get( 'minify_concatenate_css', 0 ) ) {
```

Comment:
```suggestion
		if ( ! (bool) $this->options->get( 'minify_concatenate_css', 0 ) ) {
```

We should bail-out if the option is not enabled